jannovar-core/src/test/java/de/charite/compbio/jannovar/reference/AminoAcidChangeTest.java

```diff
@@ -13,42 +13,56 @@
 
 	@Before
 	public void setUp() throws Exception {
-		this.ref = "AAACCCAAACCC";
-		this.ref2 = "CATCATCTTCA";
+		this.ref = "LLLCCCLLLCCC";
```

Comment:
This is just 'cosmetics', just making them look 'more' like AA sequences than like a nucleotide sequence (of course the previous sequence was also a valid AA sequence)